tests/sample_eventgen_conf/medium_test/eventgen.conf.syslogoutput

```diff
@@ -8,4 +8,4 @@ end = 1
 outputMode = syslogout
 syslogDestinationHost = 127.0.0.1
 syslogDestinationPort = 9999
-
+syslogAddHeader = false
```

Comment:
can we add one more test case to address this feature?
by default it is false. 
We need one more case to test if syslogAddHeader=true

Reply:
I added a new separate test for this - considered integrating it into existing one but thinking it might be confusing.